manifests/overlays/prod/applications/data_hub/dh-prod-argo.yaml

```diff
@@ -37,3 +37,17 @@ spec:
       selfHeal: true
     syncOptions:
     - Validate=false
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: obslytics.dh-prod-argo
```

Comment:
Isn't this name (as well as the file) wrong?

Reply:
Yes, you're totally right. Changed to its own file.
Also, should I create an stage application?